src/entity/posts/utils.ts

```diff
@@ -603,3 +608,26 @@ export const normalizeCollectionPostSources = async ({
 export const getPostVisible = ({ post }: { post: Pick<Post, 'title'> }) => {
   return !!post?.title?.length;
 };
+
+export const clearPostTranslations = async (
+  con: DataSource,
+  postId: string,
+  field: TranslateablePostField,
+) => {
+  if (!translateablePostFields.includes(field)) {
+    throw new Error('Invalid field');
+  }
+
+  await con
+    .getRepository(Post)
+    .createQueryBuilder()
+    .update(Post)
+    .set({
+      translation: () => `(
+SELECT jsonb_object_agg(key, value - '${field}')
+FROM jsonb_each(translation)
+)`,
```

Comment:
Because the field structure is
```json
[language]: {
  [field]: string
}
```
we need to iterate over the column and only remove the required keys, so that we don't accidentally wipe out other keys we might have in the future.